diff --git a/setup.py b/setup.py
index 01885c3..62c6d47 100644
--- a/setup.py
+++ b/setup.py
@@ -64,9 +64,10 @@ def have_openmp():
     compiler = get_compiler()
     compile_args = ["-fopenmp"]
 
-    if is_clang():
-        compile_args.insert(0, "-Xpreprocessor")
-        compile_args.append("-lomp")
+    # [conda-forge]: will not be `clang` on windows
+    # if is_clang():
+    #     compile_args.insert(0, "-Xpreprocessor")
+    #     compile_args.append("-lomp")
 
     if have_homebrew():
         compile_args.extend(get_homebrew_include())
